README.md

```diff
@@ -10,6 +10,11 @@ Things you may want to cover:
 * System dependencies
 
 * Configuration
+Remember to add to your credentials:
+devise_secret_key: { generated with rails secret }
+api_client_key: { generated with rails secret }
+api_valid_jwt: { generated using API::V1::Authentication.api_jwt } Expires in two weeks!
```

Comment:
I'm not sure I understand why we need this and what happens when it expires? Actually could you add to the README some requests that the client would use, using `curl`, e.g. login, update name or something, what response they get when token expired and what they need to do then - that will also make it easier for me to understand.

Reply:
Sorry for that. I thought about it so much that I got carried away by the greatness of my own logic.
I added a few notes in the readme but the basic idea is:
No authentication is needed for groups/subgroups/indicators. 
Api-Auth is required (in the headers) for signup and login using an api-jwt.
This api-jwt has an api-key encrypted inside and can be generated using a rails task.
Authentication is required (in the headers) for GET/PUT users/me.
This Authentication is a jwt with the user id encrypted inside and is retrieved in the response of signup/login

Reply:
- Don't worry about expiration because api_jwt won't expire.

Reply:
And finally answering to : do we need api_valid_jwt into credentials? Only if we want the client to use **exactly that valid token** and not  any valid token.